README.md

```diff
@@ -77,3 +80,23 @@ ansible-playbook -i galaxy/openshift-toolkit/custom-dashboards/.applier galaxy/o
 # Deploy MDT Tool
 ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e provision=false
 ```
+
+
+### Deploying Exporters
+
+To deploy the exporters, first setup the following environment variables and create the following secret:
+
+```
+export GITHUB_REPOS=redhat-cop/mdt-quickstart
```

Comment:
`GITHUB_REPOS` can be single value or a comma separated list

Reply:
Updated the documentation.